lifecycle/lifecycle.go

```diff
@@ -48,6 +48,20 @@ const (
 	Governor
 )
 
+//lint:ignore U1000 #1768 support for external clients
+var typeToString = map[Type]string{
+	Startup:     "Startup",
+	Shutdown:    "Shutdown",
+	Provisioned: "Provisioned",
+	Alive:       "Alive",
+	Governor:    "Govenor",
+}
+
+//lint:ignore U1000 #1768 support for external clients
+func (t Type) toString() string {
```

Comment:
make `String()` so just printing the type will do the right thing

Reply:
Also so that it'd actually be available externally >.<